directives/submit.py

```diff
@@ -109,6 +110,17 @@ def run(self):
             u'max_group_size': data.get('max_group_size', env.config.default_max_group_size),
             u'points_to_pass': self.options.get('points-to-pass', data.get('points_to_pass', 0)),
         })
+
+        if self.content:
+            self.assert_has_content()
+            instructions = '\n'.join(self.content)
+            try:
+                parts = publish_parts(source=instructions, writer_name='html')
+            except Exception:
+                raise self.error('Error in submit content!')
```

Comment:
The error message must contain info about the actual error instead of just saying there was some error. Something like 
```
except Exception as e:
    raise self.error(str(e))
```